package.json

```diff
@@ -4,7 +4,9 @@
     "build": "turbo run build",
     "dev": "turbo run dev",
     "lint": "turbo run lint",
+    "lint:strict": "eslint --max-warnings=0 apps/**/*.{ts,tsx}",
```

Comment:
Can we take advantage of the turbo repo ? And lint every package?

Reply:
Sure